try.html

```diff
@@ -529,6 +529,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
   <td><img src='/gem/u/raphink.svg' alt=''/></td>
   <td><code>https://img.shields.io/gem/u/raphink.svg</code></td>
   </tr>
+  <tr><th> APIstatus.org: </th>
+  <td><img src='/apistatus/o/aHR0cHM6Ly9tb2NrYmluLmNvbS9zdGF0dXMvMjAw.svg' alt=''/></td>
```

Comment:
I don't think this format will be of any use to anybody. Who would read this on the main page and guess that they have to base64 their URL to use this? Would would understand the distinction between the `o` version and the `t` version?